src/sentry/issues/issue_occurrence.py

```diff
@@ -35,7 +35,7 @@ class IssueOccurrenceData(TypedDict):
     detection_time: float
     level: str | None
     culprit: str | None
-    initial_issue_priority: NotRequired[int]
+    initial_issue_priority: NotRequired[int | None]
```

Comment:
@snigdhas do you know if `initial_issue_priority` is both NotRequired and nullable? in other places i see it only as nullable, but haven't looked exhaustively

Reply:
nullable seems incorrect fwiw

Reply:
I think that as its a newer feature, None is possible, at least i see it passed / set to None in places.

Reply:
I don't see anywhere we set it to None? NotRequired should be sufficient.

Reply:
@snigdhas https://github.com/getsentry/sentry/blob/adb2588fc573cf65e868c378f48daaa98603cfb7/src/sentry/issues/issue_occurrence.py#L143

this can make it be None, as the .get will return None if the key is not defined.

Reply:
hmm, we could fix that one instance but we'd still need a not-None default [here](https://github.com/getsentry/sentry/blob/8274baa16d6c9fff540af06eb8d8335bf1757a8c/src/sentry/issues/issue_occurrence.py#L90), which would make this backwards incompatible since we can't set a sensible default value for the priority without knowing the issue type.

Maybe it makes more sense to drop the `NotRequired` in favor of `int | None`